backend/compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: '3.8'
+
+services:
+  api:
+    env_file:
+      - .env
+    container_name: api
+    build: .
+    command: bash -c 'while !</dev/tcp/db/5432; do sleep 1; done; uvicorn main:app --proxy-headers --host 0.0.0.0 --reload'
```

Comment:
Może lepiej to wsadzić do Dockerfile'a
[Tutaj](https://github.com/RooTender/MTAS/blob/develop/src/backend/dockerfile) jest przykład mojego projektu, w którym używałem uvicorna z fastapi w ten sposób

Reply:
teoretycznie można ale z tego co wiem to jeżeli się korzysta z composa to robi się to w composie, Dockerfile służy do przygotowania tego kontenera natomiast to docker compose odpowiada z zrunnowanie. Nigdy jeszcze nie spotkałem się żeby ktoś zrobił inaczej. W projekcie, który podesłałeś nie macie żadnych innych zależności dlatego to odpalacie prosto z Dockerfila